commands/deploy.go

```diff
@@ -231,21 +228,25 @@ func addFileToTarWriter(filePath string, tarWriter *tar.Writer) error {
 		return fmt.Errorf(fmt.Sprintf("Could not get stat for file '%s', got error '%s'", filePath, err.Error()))
 	}
 
-	header := &tar.Header{
-		Name:    filePath,
-		Size:    stat.Size(),
-		Mode:    int64(stat.Mode()),
-		ModTime: stat.ModTime(),
+	header, err := tar.FileInfoHeader(stat, filePath)
+	if err != nil {
+		return err
 	}
 
+	// must provide real name
+	// (see https://golang.org/src/archive/tar/common.go?#L626)
+	header.Name = filepath.ToSlash(filePath)
```

Comment:
`filepath` package versus `filePath` variable looks like a recipe for future maintenance headaches. Consider not depending on case as the only difference.